application/views.py

```diff
@@ -56,12 +57,9 @@ def save_existing_application(request, application_id):
             f.parent_name = form.parent_name
             f.parent_phoneNumber = form.parent_phoneNumber
             f.submitted_date = timezone.now()
-            try:
-                if "Submit" in request.POST.get("submit"):
-                    f.is_draft = False
-                else:
-                    f.is_draft = True
-            except:  # noqa: E722
+            if request.POST.get("submit") is not None:
```

Comment:
Will we not get a KeyError if the "submit" key doesn't exist in the request session? Just curious since I'm not certain about this.

Reply:
No it is checked to None. If it was request.POST["submit"] then would have got that error.